midi_app_controller/actions/napari_actions.py

```diff
@@ -175,12 +197,12 @@ def increase_gamma(ll: LayerList) -> None:
     Action(
         id="napari:viewer:increase_dimensions_left",
         title="Increase dimensions to the left",
-        callback=increase_dimensions_left,
+        callback=partial(increment_axis, slider_num=None, inc=-1),
```

Comment:
In a follow-up PR I want to be able to assign actions permanently to specific sliders, rather than the last num. Or do you mean in this particular line? For that I just thought it would be nice to be consistent everywhere.